### DISCUSSION TITLE
Access Hyperlink value from Entity

### DISCUSSION THREAD

**Discussion by tomasmcguinness** [DomCR/ACadSharp#376] [Q&A]
Hello,

I'm looking to access the Hyperlink value that is set on an Entity.

![image](https://github.com/DomCR/ACadSharp/assets/302741/4a6fb2ac-06da-4626-b838-b5c288ef0c7e)

I can't see this defined anywhere (aside from the HyperlinkBase on the document)

It this available or do I need to customize the code?

**Comment by DJGosnell** [comment]
I actually implemented this previously on another project.  It has to do with the XData.  See below.

https://github.com/haplokuon/netDxf/blob/7d45abe7b3d5bdc4db35389114a87535093c0403/netDxf/Entities/EntityObject.cs#L253-L315

**Reply by tomasmcguinness** [reply]
Brillant. Thanks! 

It's a great library too, so thanks for the hard work!
